scripts/create-client-mock-data.js

```diff
@@ -13,6 +17,24 @@ async function main() {
 
   const mockData = JSON.parse(data)
 
+  mockData.footprint.forEach((footprint) => {
+    let updatedMonthStr = updatedMonth.toISOString()
+    footprint.timestamp = updatedMonthStr
+    updatedMonth = getPreviousMonth(updatedMonth)
+    updatedMonth.setUTCHours(0, 0, 0, 0)
+  })
+
+  fs.writeFile(
+    path.resolve(__dirname, `../packages/client/stub-server/mockData.json`),
+    JSON.stringify(mockData),
+    (err) => {
+      if (err) {
+        console.error(err)
+        return
+      }
+    },
+  )
```

Comment:
Unneeded return statement on line 33, as it will automatically stop after the log without it when an error occurs. 

I would also recommend using `fs.writeFileSync` function instead. The normal `writeFile` is an asynchronous function that will continue running in the background as the rest of the script is running. Writing the file synchronously will be a safer way since the rest of the code relies on it.